src/Common/src/Interop/Windows/mincore/Interop.GetFullPathNameW.cs

```diff
@@ -9,10 +10,30 @@ partial class Interop
 {
     partial class mincore
     {
+        /// <summary>
+        /// WARNING: This overload does not implicitly handle long paths.
+        /// </summary>
```

Comment:
Let's rename it then. It's strange to have two overloads so similar and yet with so different behavior.